server/node.go

```diff
@@ -128,40 +123,51 @@ func BootstrapCluster(clusterID string, eng engine.Engine, stopper *util.Stopper
 	localDB := client.NewKV(nil, kv.NewTxnCoordSender(lSender, ctx.Clock, false, stopper))
 	ctx.DB = localDB
 	ctx.Transport = multiraft.NewLocalRPCTransport()
-	// The bootstrapping store will not connect to other nodes so its StoreConfig
-	// doesn't really matter.
-	s := storage.NewStore(ctx, eng)
+	for i, eng := range engines {
```

Comment:
I don't believe this is the correct thing to do. Only a single store should ever be bootstrapped this way.